### PR TITLE
Fix/uche/code clean up

### PR DESCRIPTION
**Pull request by Way2nnadi**

### CODE REVIEW

File: src/components/aboutUs/components/TeamProfile.js [hunk 1]
```diff
@@ -4,37 +4,36 @@ import { Avatar, Card, Icon, List } from 'antd';
...
         return(
-            <List  
-                bordered={false}
+            <List
                 dataSource={TEAM_INFO}
-                grid={{ gutter: 12, lg: 4, md: 1.7, sm: 1.8, xs: 1.7, xxl: 3 }}
-                renderItem={item => (
-                    <List.Item>   
+                header={<h4 className={TEAM_HEADER}>{HEADER_TEXT}</h4>}
+                grid={{ column: 3, lg: 3, md: 1, sm: 1, xs: 1, xxl: 3 }}
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/TeamProfile.js]: Do not use decimal places when dealing with a grid. Only whole numbers.

File: src/components/aboutUs/components/TeamProfile.js [hunk 2]
```diff
@@ -4,37 +4,36 @@ import { Avatar, Card, Icon, List } from 'antd';
...
 const actionList = SOCIAL_MEDIA.map(social => (<Icon type={social} key={social} />));
 
 class TeamProfile extends React.Component {
     render() {
         return(
-            <List  
-                bordered={false}
+            <List
                 dataSource={TEAM_INFO}
-                grid={{ gutter: 12, lg: 4, md: 1.7, sm: 1.8, xs: 1.7, xxl: 3 }}
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/TeamProfile.js]: Do not use decimal places when dealing with a grid. Only whole numbers.

Comment (EngrEric) [reply src/components/aboutUs/components/TeamProfile.js]: why is decimal not used for gird?

Comment (joshuaobinna) [reply src/components/aboutUs/components/TeamProfile.js]: yea

**Comment by joshuaobinna** [comment]
looks good to me

File: src/components/aboutUs/components/TeamProfile.js (hunk 2 above)
Comment (Way2nnadi) [reply src/components/aboutUs/components/TeamProfile.js]: `...lg: 4, md: 1.7, sm: 1.8, xs: 1.7, xxl: 3 }`
so what is this piece of code saying? from left to right, it is saying for large screens show 4 items, for middle sized screens show 1.7 items, ...for extra extra large screen show 3 items. You can't have 1.7 items, so what does the browser do, it adds extra white space to met the request that you made of it. Which leads to messed up UI

**Pull request merged**